examples/device/cdc_dual_ports/src/main.c

```diff
@@ -30,13 +30,17 @@
 
 #include "bsp/board.h"
 #include "tusb.h"
+#include "stm32h7xx_hal.h"
+#include "stm32h7xx.h"
 
 //------------- prototypes -------------//
 static void cdc_task(void);
 
 /*------------- MAIN -------------*/
 int main(void)
 {
+  SCB->VTOR = 0x08040000;
```

Comment:
is it really needed, the start up/system init would probably does this for us. I will look closer at this if you are willing to test out my update.

Reply:
So So sorry I became incredibly sick, But Im mostly back at it now! Thank you guys so much! This project has the best community! Ok So I made sum mistakes, I didnt know any and all git pushes I would do would immediately be added to PR so Things I didnt intend to make it hear did, Im fairly embarrassed. So the vector offset I will hafta try and see if it works without it being specifically listed under main. I currently have several stm32s with boot loaders that all seem to require different levels of offset configs. Most seem fine with .ld having the base address for flash set at offset, sum require the first line of main, (and others something in between) never been smart enuff to figure out why. But I will test.  Thank you again! This is an awesome project and your kindness on offering to help me has not gone unnoticed. I see you ask me to try your update, Im a lil confused do you mean you have a branch I should pull and try? Or try putting the offset in the init like you mentioned? So sorry I didnt understand, but yes I would love to test any updates

Reply:
no problem at all, I am a bit busy as well. 

> I see you ask me to try your update, Im a lil confused do you mean you have a branch I should pull and try? Or try putting the offset in the init like you mentioned? So sorry I didnt understand, but yes I would love to test any updates

Thanks for your kind words and willingness to test it out, I haven't made any changes just yet. But will try to make some changes soon enough.